Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/DirectoryInitialization.java

```diff
@@ -153,6 +151,30 @@ private static void extractSysDirectory(Context context)
     // Let the native code know where the Sys directory is.
     sysPath = sysDirectory.getPath();
     SetSysDirectory(sysPath);
+
+    File driverDirectory = new File(context.getFilesDir(), "GPUDrivers");
+    if (!driverDirectory.isDirectory())
+    {
+      driverDirectory.mkdirs();
+    }
+    File driverExtractedDir = new File(driverDirectory, "Extracted");
+    if (!driverExtractedDir.isDirectory())
+    {
+      driverExtractedDir.mkdirs();
+    }
+    File driverTmpDir = new File(driverDirectory, "Tmp");
+    if (!driverTmpDir.isDirectory())
+    {
+      driverTmpDir.mkdirs();
+    }
+    File driverFileRedirectDir = new File(driverDirectory, "FileRedirect");
+    if (!driverFileRedirectDir.isDirectory())
+    {
+      driverFileRedirectDir.mkdirs();
+    }
```

Comment:
Instead of having all these manual checks, you should be able to use `Files.createDirectories`.

Reply:
Java.nio is only supported on Android 13 and higher. Is this important enough that I have to setup API desugaring?

Reply:
mkdirs checks whether the directory already exists too so I'll just get rid of the branches.